app/assets/javascripts/landmarks.js

```diff
@@ -0,0 +1,12 @@
+function initMap(lat, lng) {
+    var myCoords = new google.maps.LatLng(lat, lng);
+    var mapOptions = {
+    center: myCoords,
+    zoom: 17
+    };
+    var map = new google.maps.Map(document.getElementById('map'), mapOptions);
```

Comment:
The map is not working in my localhost (see my Slack message / screenshot). Let's make sure everything is configured right before we merge this / push to Heroku.

Reply:
Try reloading landmarks table.

Reply:
Map works after running updated rake task!